Gruntfile.js

```diff
@@ -452,7 +477,7 @@ module.exports = (grunt) => {
   })
 
   grunt.registerTask('build', function () {
-    const esbuild = this.flags.watch ? 'esbuild:watch' : 'esbuild'
+    const esbuild = this.flags.deploy ? 'esbuild:deploy' : (this.flags.watch ? 'esbuild:watch' : 'esbuild')
```

Comment:
It appears this `esbuild:deploy` flag is not used anywhere, so it seems like both `esbuild:deploy` and `build:deploy` flags can be removed?

Reply:
Nice catch!